public/js/all.js

```diff
@@ -376,3 +376,21 @@ function initializeTextareaCounter() {
 }
 
 window.addEventListener('load', initializeTextareaCounter);
+
+let shouldShowTopFade = false;
+let shouldShowBottomFade = true;
```

Comment:
@EmoonX Right now this is global I think. Not sure if that matters but something to consider.